cmd/nerdctl/network_rm_linux_test.go

```diff
@@ -49,3 +49,20 @@ func TestNetworkRemove(t *testing.T) {
 	_, err = netlink.LinkByName("br-" + networkID[:12])
 	assert.Error(t, err, "Link not found")
 }
+
+func TestNetworkRemoveWhenLinkWithContainer(t *testing.T) {
+	t.Parallel()
+	if rootlessutil.IsRootless() {
+		t.Skip("test skipped for remove rootless network")
+	}
+	base := testutil.NewBase(t)
+	networkName := testutil.Identifier(t)
```

Comment:
replace by `tID := testutil.Identifier(t)` and use it for container and network as name
then delete this https://github.com/containerd/nerdctl/pull/1569/files#diff-b72bcbd8341534dc0935d345433cc56537515e91cf077d030cb817beb84a9971R64

Reply:
But I think https://github.com/containerd/nerdctl/blob/main/pkg/testutil/testutil.go#L606-L616 we have different case name here.

Reply:
@Zheaoli  why do you mean by `we have different case name here` ? 
`network` and `container` can have the same name, right ? the idea is to reduce var allocation in the same test case

Reply:
Ohhhh, I got your means, although I kept the same with the previous test. I will update them both

Reply:
Done